cmd/update/update_scheduler.go

```diff
@@ -0,0 +1,157 @@
+// maestro-cli
+// https://github.com/topfreegames/maestro-cli
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package update
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/spf13/cobra"
+	"github.com/topfreegames/maestro-cli/common"
+	"github.com/topfreegames/maestro-cli/extensions"
+	"github.com/topfreegames/maestro-cli/interfaces"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	v1 "github.com/topfreegames/maestro/pkg/api/v1"
+	yaml "gopkg.in/yaml.v2"
+	k8s_yaml "sigs.k8s.io/yaml"
+)
+
+var marshler = &runtime.HTTPBodyMarshaler{
```

Comment:
@caiordjesus typo here, marshler <-> marshaler

Reply:
To be honest i'm a bit bugged hahahaha
When searching on the internet I actually found out that the name should be Marshaller, not marshaler
Not only that but the IDE actually creates a typo problem and fixes it with "Marshaller". I'll let this name then lol